### PR TITLE
Slider component

### PR DESCRIPTION
**Pull request by dev-enforced**
This Pull Request has the slider component and the utilities available for flex and position properties.

### CODE REVIEW

**Review by SaurabhChawda — APPROVED** [review]
-Code looks good to me.
-Liked your Color selection and presentation.

File: components/sliders.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    opacity:1;
+}
+
+.gentle-input-slider::-webkit-slider-thumb {
+    cursor: pointer;
+    -webkit-appearance: none;
+    appearance: none;
+    width: 10px;
+    height: 20px;
+    background:#fff;
```
Comment (ozairkhan2810) [inline components/sliders.css]: can use color variable

File: style-guide/flex-position-utilities.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+.fixed-position{
+    position: fixed;
+}
+
+.g-flex-row{
+    display:flex;
+}
+
+.g-flex-column{
+    display: flex;
```
Comment (ozairkhan2810) [inline style-guide/flex-position-utilities.css:19]: can group flex property

**Review by ozairkhan2810 — APPROVED** [review]
couple of suggestions, rest looks good
Class names are meaningful
Good to merge

File: components/sliders.css (hunk 1 above)
Comment (dev-enforced) [reply components/sliders.css]: Yes using it

**Pull request merged**